src/instructlab/config.py

```diff
@@ -30,7 +30,7 @@
 DEFAULT_NUM_INSTRUCTIONS = 100
 DEFAULT_PROMPT_FILE = "prompt.txt"
 DEFAULT_GENERATED_FILES_OUTPUT_DIR = "generated"
-DEFAULT_CONNECTION_TIMEOUT = httpx.Timeout(timeout=30.0)
+DEFAULT_CONNECTION_TIMEOUT = httpx.Timeout(timeout=300.0)
```

Comment:
That's going to cause issue if the client tries to connect to the server (during a chat session) and the latter crashed, users will wait for 5min... Can we change the client timeout for the user case (train?) that needs 5min?

Reply:
are you sure? I would expect the connection to be immediately rejected if there's not a process listening on that port anymore.

Reply:
either way it's not clear why this was necessary, so I'll drop it from the PR for now